server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -65,7 +65,7 @@ public void testPrimaryStopped_ReplicaPromoted() throws Exception {
         assertHitCount(client(replica).prepareSearch(INDEX_NAME).setSize(0).setPreference("_only_local").get(), 2);
 
         // assert we can index into the new primary.
-        client().prepareIndex(INDEX_NAME).setId("3").setSource("bar", "baz").setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE).get();
+        client().prepareIndex(INDEX_NAME).setId("3").setSource("bar", "baz").setRefreshPolicy(WriteRequest.RefreshPolicy.WAIT_UNTIL).get();
```

Comment:
How does changing from `IMMEDIATE` to `WAIT_UNTIL` fixes the flakyness ? The`IMMEDIATE` refresh policy forces refresh with indexing request which means doc should already be searchable.

Reply:
You are right it doesn't, IMMEDIATE blocks until a refresh completes, wait_until waits until the next refresh cycle to complete.  Neither of these was the reason for flakiness, removed this change.

Reply:
Updated PR description for when this test was fixed, this is simply to unmute it.

Reply:
Thanks @mch2 for confirmation.  I see this was muted in https://github.com/opensearch-project/OpenSearch/pull/5898 without any description (though I know it failed once in a while before).